### ISSUE TITLE
Missing supporter benefits info

### ISSUE THREAD

**Issue by Borderpatrol1987** [Clon1998/mobileraker#414]
### Description
When you pull up supporter perks from the printer config page, it doesn't load what the perks are on the sign-up page. 

### Steps to Reproduce
Go to printer configuration 
Click on the hand icon next to printer theme
Click the "!“ next to subscriber perks 

### Expected Behavior
A popup with info should appear like when accessing from app settings. 

### Version Information

- Device-OS (Android/iOS): [OS] Android 14
- Mobileraker: [version] 2.7.5

**Comment by monterman** [comment]
me too,  also in android 14 does not retain log in session for lifetime support tier, defaulting to free version every time app restarts or unloads from memory.

**Comment by Clon1998** [comment]
> me too,  also in android 14 does not retain log in session for lifetime support tier, defaulting to free version every time app restarts or unloads from memory. 

I assume you recently migrated to a new device?

Please check this issue:

https://github.com/Clon1998/mobileraker/issues/543#issuecomment-3924046555

TLDR: Backup the app data, remove the app completely reinstall it. If it's not working disable Google Backups for it and try it again.

**Comment by monterman** [comment]
It's worked, thanks. Had to remove Google backups. Uninstall, reboot, imported old configs.

Yahoo Mail: Search, Organize, Conquer 
 
  On Tue, Apr 21, 2026 at 2:28 PM, Patrick ***@***.***> wrote:   Clon1998 left a comment (Clon1998/mobileraker#414)

me too, also in android 14 does not retain log in session for lifetime support tier, defaulting to free version every time app restarts or unloads from memory.


I assume you recently migrated to a new device?

Please check this issue:

#543 (comment)

TLDR: Backup the app data, remove the app completely reinstall it. If it's not working disable Google Backups for it and try it again.

**Comment by Clon1998** [comment]
Seems like I forgot to close that while fixing the issue. Also the system was reworked so it should not be present anymore!

Thanks for reporting the issue tho!
